website/app/styles/markdown/tables.scss

```diff
@@ -33,4 +32,21 @@ $doc-markdown-table-border-color: rgba(101, 106, 118, 20%);
   .doc-markdown-tr:last-child & {
     border-bottom: none;
   }
+
+  code {
```

Comment:
[issue] these styles work for our specific use case, but may impact other cases when these HTML elements are used inside a generic markdown table. probably better to create a specific CSS file for the `doc-component-api` component, and use the class `.doc-component-api` to target these (even if they are temporary).

Reply:
leftover from when playing with markdown table – will remove